### PR TITLE
I add components for initial Roadmap screen.

### PR DESCRIPTION
**Pull request by sweetmantech**
# RoadMap Page
Screenshot 2023-01-29 at 6 01 38 PM

### CODE REVIEW

File: components/Roadmap/Roadmap.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+  }, [])
+
+  useEffect(() => {
+    window.scrollTo(scrollPosition.x, scrollPosition.y)
+  }, [scrollPosition])
+
+  return (
+    <div className="sm:h-screen h-[200vh] w-[200vw] flex items-center  gap-3 pl-[154vw] sm:pl-[50vw] overflow-x-scroll">
+      <button className={className} onClick={() => setActiveStep(4)}>
```
Comment (crazzywizard) [inline components/Roadmap/Roadmap.tsx]: might as well just use a map function, a lot repeated code.
something like 
```
interface RoadmapCard = {
 stepNumber: integer;
className: String;
displayText:String
}
const RoadmapCards:RoadmapCard[] = [{},{},{},{]]
```

File: components/Roadmap/Roadmap.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        Watch Trailer
+      </button>
+      <button className={className} onClick={() => setActiveStep(4)}>
+        Mysteries Revealed
+      </button>
+      <button className={className} onClick={() => setActiveStep(4)}>
+        Storyline
+      </button>
+      <button className={className} onClick={() => setActiveStep(4)}>
+        Play Whitelist Quiz
```
Comment (crazzywizard) [inline components/Roadmap/Roadmap.tsx]: so these are extra large buttons. Interesting to go with large buttons rather than clickable cards.

Comment (sweetmantech) [reply components/Roadmap/Roadmap.tsx]: i prefer clickable cards, any tips?

File: components/Roadmap/Roadmap.tsx (hunk 1 above)
Comment (sweetmantech) [reply components/Roadmap/Roadmap.tsx]: updated to use `map` ✅ 
Screenshot 2023-02-03 at 10 02 52 AM

File: components/Roadmap/Roadmap.tsx (hunk 2 above)
Comment (sweetmantech) [reply components/Roadmap/Roadmap.tsx]: updated to use `<div />` instead of `<button />`
Screenshot 2023-02-03 at 10 22 07 AM

**Pull request merged**